### PR TITLE
Fix submit form

### PR DESCRIPTION
**Pull request by julieg18**
* lower input clear tab index
* mark input fields as error on blur
* add hover style to form close button
* remove scroll from phone form popup

### CODE REVIEW

File: src/components/FormInput/FormInput.js [hunk 1]
```diff
@@ -35,18 +33,25 @@ export default function FormInput({
...
   }
 
-  function handleInputBlur() {
+  function handleInputBlur(e) {
+    const input = e.target;
+    const isValid = input.validity.valid;
     setIsInputFocused(false);
+    setIsInputValid(isValid);
+    setIsInputCaptionVisible(input.value !== '');
+    setValidationMessage(input.validationMessage);
```
Comment (iceicery) [inline src/components/FormInput/FormInput.js:54]: Nice job moving around codes between blur, focus and change.  It fixed the issue perfectly.

**Review by iceicery — COMMENTED** [review]
Great job fixing the issues. No errors at this update!! I notice there is only one thing left addressed at form issues: Remove the inner shadow from the disabled states for buttons (design oversight, fixed). Sorry I didn't realize this while fixing the button component. Do you want to include at this PR?

**Comment by julieg18** [comment]
Sure, but I probably won't be able to get on my computer till tonight or tomorrow morning.

**Comment by iceicery** [comment]
> Sure, but I probably won't be able to get on my computer till tonight or tomorrow morning.
 I am at my computer. How about I approving your PR and create another one?

**Comment by julieg18** [comment]
Sounds good!

**Review by iceicery — APPROVED** [review]
Great job!

**Pull request merged**